src/types.ts

```diff
@@ -52,6 +58,14 @@ export interface StringMap {
 
 export type Choice = "call" | "invoke";
 
+export function extractTxHash(response: string) {
+    return extractFromResponse(response, /^Transaction hash: (.*)$/m);
+}
+
+export function extractAddress(response: string) {
```

Comment:
Does this need to be exported? And why were these two extract functions moved up a few lines?